### ISSUE TITLE
hamburger icon not visible when in dark mode and small window size

### ISSUE THREAD

**Issue by touseefbsb** [Microsoft/WindowsTemplateStudio#2143]
## For Bugs:

in navview template when we turn dark mode from settings and resize the window to a smaller window the top bar is of white color, where it should be dark. and the hamburger menu should be white, and I think it is white that is why it isn't visible. bcz I can click there and hamburger works as expected.

![whitehamburgermenu](https://user-images.githubusercontent.com/11542189/38391749-8d39fa5a-393e-11e8-94ba-4c4630a7e78d.PNG)

in the screenshot above I have circled the area where the icon should be.
## System

* **VS Version:** 15.6.5
* **Windows Build:** RS4

<genTemplate:Metadata>
    <genTemplate:Item Name="generator" Value="Windows Template Studio" />
    <genTemplate:Item Name="wizardVersion" Version="v0.17.18089.1" />
    <genTemplate:Item Name="templatesVersion" Version="v0.17.18089.1" />
    <genTemplate:Item Name="projectType" Value="SplitView" />
    <genTemplate:Item Name="framework" Value="MVVMBasic" />
</genTemplate:Metadata>

**Comment by crutkas** [comment]
@touseefbsb Your on an insider flight, which exact build?

@sibille we should verify on the navview transition this works.

**Comment by touseefbsb** [comment]
build  : 17133.1

**Comment by mrlacey** [comment]
The branch in question has already been merged and running from dev branch does not show this behavior.
Checked from running on dev, not checked on nightly.

**Comment by touseefbsb** [comment]
@mrlacey  is there a simpler way where I can fix this issue in my project? without having to generate a new wts project?

**Comment by mrlacey** [comment]
@touseefbsb To work around this you'll need to change the way the toolkit swaps out the hamburgermenu control. It doesn't support a way of modifying the appropriate template itself.

If your min targetting SDK will be FCU then you can switch to just use that ([docs](https://docs.microsoft.com/en-us/windows/uwpcommunitytoolkit/controls/hamburgermenu#navview)). If you need to keep supporting the Creators Update, you can use the workaround at: https://github.com/Microsoft/WindowsTemplateStudio/issues/1763#issuecomment-363251560

**Comment by touseefbsb** [comment]
@mrlacey  as per ur instructions I used the workaround suggestion in the comment, everything was working fine, as per that comment, but now to be sure whether that work around fixes this issue I went to dark mode again and noticed the issue remains exactly the same, still I cant see a hamburger icon in the dark mode in short sized window.

**Comment by touseefbsb** [comment]
@mrlacey  even after applying the workaround the issue remains exactly the same, so as u said that this is the default behaviour of the navigationview ( which seems like a bug in the default control ) how to change this default behaviour? from what I gather the problem isn't color of the hamburger button, bcz it looks fine , its white as it should be, the problem is the background color of the top bar we get on small size, which should be black-ish in the dark theme. but is white.

**Comment by mrlacey** [comment]
@touseefbsb using my workaround from before I can avoid the behavior you're seeing by setting the background of the containing Grid to a theme aware brush:

```
<Grid Background="{ThemeResource SystemControlBackgroundAltHighBrush}">
    <controls:HamburgerMenu
```

**Comment by touseefbsb** [comment]
@mrlacey  if this can be fixed with changing the background of top level grid, then why do we needed to do that workaround to swap out hamburger menu with navigationView?

**Comment by touseefbsb** [comment]
ok so technically we don't need that workaround for my issue, but that workaround will be helpful if I want to edit the header of the navigationview right?

**Comment by touseefbsb** [comment]
btw thnks, putting the background solved my issue :)

**Comment by mrlacey** [comment]
Reopening as this needs verifying with the 2.0 templates.
The issue went unnoticed in the 1.7 templates and we want a fix in the shipped templates.

**Comment by touseefbsb** [comment]
yeah I hope its fixed in 2.0 :)

**Comment by sibille** [comment]
Checked in Navview templates, icon is showing correctly. 
Checked in current Hamburger Menu templates, Hamburger icon is not showing correctly. Will fix for 2.0

**Comment by mvegaca** [comment]
Fix added in master branch on PR #2152

**Comment by sibille** [comment]
Verified on pre-release:
Template version: 0.12.18100.2
Wizard version: 0.12.18100.2
